### PR TITLE
Fix base operator name

### PR DESCRIPTION
**Pull request by viroel**
Issue when testing operators:
`[[ keystone == \K\E\Y\S\T\O\N\E ]]`

### CODE REVIEW

**Comment by sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-keystone-operator-master-keystone-operator-build-deploy

**Comment by sandeepyadav93** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-keystone-operator-master-keystone-operator-build-deploy

keystone container was missing, reran github action - passed now

**Comment by sandeepyadav93** [comment]
The hash with which building container is different from hash which we are trying to check later.

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/36513/rehearse-36513-pull-ci-openstack-k8s-operators-keystone-operator-master-keystone-operator-build-deploy/1626772692831047680/artifacts/keystone-operator-build-deploy/openstack-k8s-operators-build/build-log.txt

~~~
The resulting image will be pushed with Docker to "quay.rdoproject.org/openstack-k8s-operators/keystone-operator-index:0a33474d8a1c3255de5d199af819eff680cd215c""
~~~

~~~
++ curl -s https://quay.rdoproject.org/api/v1/repository/openstack-k8s-operators/keystone-operator-bundle/tag/
++ jq -r '.tags[].name'
++ sort -u
++ grep 22a34597a601
~~~

**Comment by viroel** [comment]
> The hash with which building container is different from hash which we are trying to check later.
> 
> https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/36513/rehearse-36513-pull-ci-openstack-k8s-operators-keystone-operator-master-keystone-operator-build-deploy/1626772692831047680/artifacts/keystone-operator-build-deploy/openstack-k8s-operators-build/build-log.txt
> 
> ```
> The resulting image will be pushed with Docker to "quay.rdoproject.org/openstack-k8s-operators/keystone-operator-index:0a33474d8a1c3255de5d199af819eff680cd215c""
> ```
> 
> ```
> ++ curl -s https://quay.rdoproject.org/api/v1/repository/openstack-k8s-operators/keystone-operator-bundle/tag/
> ++ jq -r '.tags[].name'
> ++ sort -u
> ++ grep 22a34597a601
> ```

Yeah, this is not correct. Since this is a rehearsal job, we can consume bundles from original registry. That's why hash is not matching, since we didn't push the latest hash (from keystone for instance) to rdoquay.

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-keystone-operator-master-keystone-operator-build-deploy

**Comment by viroel** [comment]
keystone-operator-build-deploy job is now passing https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/rehearse-36513-pull-ci-openstack-k8s-operators-keystone-operator-master-keystone-operator-build-deploy

**Comment by sandeepyadav93** [comment]
/lgtm

thanks for fixing the job.

File: ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh [hunk 1]
```diff
@@ -126,6 +126,12 @@ if [[ "$BASE_OP" != "$META_OPERATOR" ]]; then
...
     REPO_NAME=${PR_REPO_NAME}
+    # NOTE(dviroel): We need to replace registry in bundle only when testing
+    #  a PR against operator's repo. When testing rehearsal jobs, we consume
+    #  from latest commit.
+    export IMAGENAMESPACE=${ORGANIZATION}
+    export IMAGEREGISTRY=${REGISTRY}
+    export IMAGEBASE=${SERVICE_NAME}
   fi
 
   go mod edit -replace github.com/${ORG}/${BASE_OP}/api=github.com/${REPO_NAME}/api@${API_SHA}
```
Comment (sandeepyadav93) [inline ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh:137]: Another optimization is to add a condition to only run the `go mod` edit for IS_REHEARSAL=false case.

**Pull request merged**